tests/test_pca.py

```diff
@@ -551,6 +551,23 @@ def test_pca_layer():
     np.testing.assert_equal(X_adata.varm["PCs"], layer_adata.varm["PCs"])
 
 
+@needs.dask
+def test_covariance_eigh_impls():
+    adata_mem = pbmc3k_normalized()
+    adata_dask = adata_mem.copy()
+    adata_dask.X = DASK_CONVERTERS[_helpers.as_sparse_dask_array](adata_dask.X)
+
+    sc.pp.pca(adata_mem, svd_solver="covariance_eigh")
+    sc.pp.pca(adata_dask, svd_solver="covariance_eigh")
+
+    assert (
+        np.linalg.norm(
+            np.abs(adata_mem.obsm["X_pca"]) - np.abs(adata_dask.obsm["X_pca"].compute())
+        )
+        < 2e-05
+    )
```

Comment:
Why do we do this `np.abs` business? Why not just check their closeness?

Reply:
Math.

- `abs`: I think it’s mathematically equivalent to flip the components because they’re linearly independent.
- `norm`: no clue, but maybe some other way the PCA components can be mathematically equivalent while not being numerically identical

Reply:
hm, maybe the norm isn’t necessary, I’ll try without